camel/utils/token_counting.py

```diff
@@ -244,7 +244,12 @@ def count_tokens_from_messages(self, messages: List[OpenAIMessage]) -> int:
                             )
                         elif item["type"] == "image_url":
                             image_str: str = item["image_url"]["url"]
-                            detail = item["image_url"]["detail"]
+                            if "detail" in item["image_url"]:
+                                detail = item["image_url"]["detail"]
+                            else:
+                                # Automatically set the detail type if the image is segmented by video.
+                                detail = OpenAIImageDetailType.AUTO
+
```

Comment:
the video content will be passed as image_url in `OpenAIMessage`, seems no need to modify the code here